src/lib.rs

```diff
@@ -25,14 +25,14 @@ type HmacSha512 = Hmac<Sha512>;
 
 #[cfg(not(feature = "cbindgen"))]
 thread_local! {
-    static LAST_ERROR: RefCell<Option<Box<Error>>> = RefCell::new(None);
+    static LAST_ERROR: RefCell<Option<Box<dyn Error>>> = RefCell::new(None);
```

Comment:
Out of sheer curiosity because I've never had a chance to use it yet, why is this now dynamically dispatched? Was a new `Error` impl added and now they need to be interchangeable?

Reply:
This was just to fix a warning: https://github.com/brave-intl/challenge-bypass-ristretto-ffi/issues/55

I think the underlying behavior is unchanged - trait objects were always dynamically dispatched, it's just that the syntax was changed to be more explicit:
https://doc.rust-lang.org/edition-guide/rust-2018/trait-system/dyn-trait-for-trait-objects.html

It's been a while, but I think I used a trait object here since I just was being a bit lazy and wanted to avoid having to define a new top level error type in this FFI crate lol.

In general I'm pretty unsatisfied with the approach I took for error handling here. It's all too easy to forget about checking for errors when we're using it in practice within Chromium ( where we don't end up raising exceptions. ) I think if I were to do this again I might define the top level "result" enum ( Ok + all error variants ) with `#[repr(C)]`, passing it across the C FFI and use cbindgen to generate a matching enum definition in the C header. That at least matches the C++ error handling we're doing elsewhere - a far cry from rust, but what can you do. ¯\\_(ツ)_/¯